app/views/keywords/create.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Result for <%= @keyword %></h1>
```

Comment:
@malparty According to our compass conventions, we should not use instance variables in views. Instead pass them as local variables from controllers:

```
   render :create, keyword: @keyword, raw_response: @raw_response
```

Reply:
Oh yeah! Thanks!